node/config/types.go

```diff
@@ -28,6 +28,7 @@ type FullNode struct {
 	Chainstore Chainstore
 	Cluster    UserRaftConfig
 	ActorEvent ActorEventConfig
+	Fevm       FevmConfig
```

Comment:
Can we call this FilecoinEVM? `FEVM` is a cute pet name we've been using, but the clearer the better in config files.